### ISSUE TITLE
latent_att not defined before. 

### ISSUE THREAD

**Issue by shreya-cmd** [eriklindernoren/Action-Recognition#15]
https://github.com/eriklindernoren/Action-Recognition/blob/b43ec097ab293ac8d83aac02b5addd27686ebe8c/models.py#L68

**Comment by tomatowithpotato** [comment]
Obviously, the author did not apply Attention Module to any model, so you can run this project while ignoring this bug
but I think latent_att can be replaced by latent_repr[0]
just like 
h_t = hidden_repr[0]
hidden_att = self.hidden_attention(h_t)

**Comment by DayDayUpDYP** [comment]
same questions

**Comment by MachineLearning11** [comment]
请问是如何解决的，我看到许多代码都没有管理这个问题，希望可以得到解决？
How to solve this problem? I see that many codes are not managed. I hope it can be solved

**Comment by MachineLearning11** [comment]
> same questions

How to solve this problem? I see that many codes are not managed. I hope it can be solved
